src/main/java/frc/robot/subsystems/Elevator.java

```diff
@@ -236,10 +249,11 @@ public static Angle heightToRotations(Distance height) {
   }
 
   public Angle ChineseRemander() {
-    double Pos3 = Math.max(0.0, Math.min(gear3.get(), 0.999)) * gear1Toothing;
-    double Pos11 = Math.max(0.0, Math.min(gear11.get(), 0.999)) * gear2Toothing;
-    return Rotations.of(
-        Constants.ElevatorConstants.CRTA[(int) Pos3][(int) Pos11] + Pos3 - (int) Pos3);
+    return Rotations.zero();
+    // double Pos3 = Math.max(0.0, Math.min(gear3.get(), 0.999)) * gear1Toothing;
+    // double Pos11 = Math.max(0.0, Math.min(gear11.get(), 0.999)) * gear2Toothing;
+    // return Rotations.of(
+    //     Constants.ElevatorConstants.CRTA[(int) Pos3][(int) Pos11] + Pos3 - (int) Pos3);
```

Comment:
I think the new version with limits should avoid the crashes. Would at least like to try it before commenting it out again.